### PR TITLE
Added infoHub to topbar

### PR DESCRIPTION
**Pull request by Krokop**

### CODE REVIEW

File: ckanext/dgua/templates/header.html [hunk 1]
```diff
@@ -45,6 +45,9 @@
                                             ('forum_index', _('Forum'))
                                         ) }}
                                     {% endblock %}
+                                    <li>
+                                        <a href="/pages/infohub">{% trans %}infoHub{% endtrans %}</a>
```
Comment (DaleDMT) [inline ckanext/dgua/templates/header.html]: url_for() is the better way, I think

File: ckanext/dgua/templates/header.html [hunk 2]
```diff
@@ -133,6 +136,9 @@ <h1>Портал відкритих даних</h1>
                                             ('forum_index', _('Forum'))
                                         ) }}
                                 {% endblock %}
+                                <li>
+                                    <a href="/pages/infohub">{% trans %}infoHub{% endtrans %}</a>
```
Comment (DaleDMT) [inline ckanext/dgua/templates/header.html]: same here

File: ckanext/dgua/templates/header.html (hunk 1 above)
Comment (e0ne) [reply ckanext/dgua/templates/header.html]: +1

Comment (Krokop) [reply ckanext/dgua/templates/header.html]: Done

**Pull request merged**